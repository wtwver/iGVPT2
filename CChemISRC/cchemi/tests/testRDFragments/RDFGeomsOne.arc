

                     SUMMARY OF PM6 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 17.163L)
                                                       Fri Jun  1 14:40:03 2018
                                                       No. of days remaining =  11

           Empirical Formula: H5 N O  =     7 atoms

PM6 GNORM=0.001 XYZ UHF CHARGE=0 Triplet

 Mopac file generated by Gabedit


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =         24.68430 KCAL/MOL =     103.27910 KJ/MOL
          TOTAL ENERGY            =       -536.02270 EV
          ELECTRONIC ENERGY       =      -1116.16133 EV
          CORE-CORE REPULSION     =        580.13863 EV
          GRADIENT NORM           =          0.06199
          DIPOLE                  =          5.36310 DEBYE    POINT GROUP:       Cs  
          (SZ)                    =          1.000000
          (S**2)                  =          2.013720
          NO. OF ALPHA ELECTRONS  =          9
          NO. OF BETA  ELECTRONS  =          7
          IONIZATION POTENTIAL    =          7.577118 EV
          ALPHA SOMO LUMO (EV)    =         -7.577  3.256
          BETA  SOMO LUMO (EV)    =        -10.113 -0.819
          MOLECULAR WEIGHT        =         35.0456
          COSMO AREA              =         81.61 SQUARE ANGSTROMS
          COSMO VOLUME            =         55.68 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     6    H     3     4.67884
            H     3    H     7     1.60659
            H     3    O     1     0.06092
          SCF CALCULATIONS        =        130
          WALL-CLOCK TIME         =          0.137 SECONDS
          COMPUTATION TIME        =          0.140 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 GNORM=0.001 XYZ UHF CHARGE=0 Triplet

 Mopac file generated by Gabedit
  O     0.67621976 +1  -1.05617698 +1   0.17399899 +1
  H     1.44866558 +1  -0.44195251 +1   0.62681043 +1
  H    -0.05206885 +1  -1.03075135 +1   0.77151181 +1
  N     3.52556529 +1   0.69056881 +1   0.91047500 +1
  H     2.28290869 +1   0.13508780 +1   0.95843946 +1
  H     3.94809441 +1   1.23733377 +1   1.63525719 +1
  H     4.11721264 +1   0.57787324 +1   0.11007452 +1
 
