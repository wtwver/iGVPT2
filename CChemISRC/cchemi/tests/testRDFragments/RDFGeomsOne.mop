PM6 GNORM=0.001 XYZ UHF CHARGE=0 Triplet

Mopac file generated by Gabedit
 O 0.001227 1 -1.571102 1 0.248322 1
 H 0.752226 1 -0.989102 1 0.248322 1
 H -0.749773 1 -0.989102 1 0.248322 1
 N 4.015299 1 1.188494 1 0.977163 1
 H 3.333780 1 0.352218 1 1.022985 1
 H 4.583026 1 1.240728 1 1.894530 1
 H 4.689215 1 1.058229 1 0.143212 1
