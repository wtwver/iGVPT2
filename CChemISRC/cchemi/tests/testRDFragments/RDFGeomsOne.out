 *******************************************************************************
 ** Site#: 1079         For non-commercial use only    Version 17.163L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 17.163L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left:  11**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Jun  1 14:40:03 2018  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  TRIPLET    - SPIN STATE DEFINED AS A TRIPLET
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *
 *
 *
 *                 CHARGE ON SYSTEM = 0
 *
 *
 *
 *  XYZ        - CARTESIAN COORDINATE SYSTEM TO BE USED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  GNORM=     - EXIT WHEN GRADIENT NORM DROPS BELOW  0.100E-02
 *******************************************************************************
PM6 GNORM=0.001 XYZ UHF CHARGE=0 Triplet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          0.00122700  *  -1.57110200  *   0.24832200  *
     2       H          0.75222600  *  -0.98910200  *   0.24832200  *
     3       H         -0.74977300  *  -0.98910200  *   0.24832200  *
     4       N          4.01529900  *   1.18849400  *   0.97716300  *
     5       H          3.33378000  *   0.35221800  *   1.02298500  *
     6       H          4.58302600  *   1.24072800  *   1.89453000  *
     7       H          4.68921500  *   1.05822900  *   0.14321200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O        0.0012   -1.5711    0.2483
     2         H        0.7522   -0.9891    0.2483
     3         H       -0.7498   -0.9891    0.2483
     4         N        4.0153    1.1885    0.9772
     5         H        3.3338    0.3522    1.0230
     6         H        4.5830    1.2407    1.8945
     7         H        4.6892    1.0582    0.1432

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: H5 N O  =     7 atoms



      MOLECULAR POINT GROUP   :   C1  


 TRIPLET STATE CALCULATION


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    9
                           NO. OF BETA  ELECTRONS =    7

          Geometry optimization using EF

  GNORM HAS BEEN SET TOO LOW, RESET TO 0.01

  SPECIFY LET AS KEYWORD TO ALLOW GNORM LESS THAN 0.01

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:   153.465 HEAT:  57.50687
 CYCLE:     2 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    80.839 HEAT:  37.96611
 CYCLE:     3 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    46.372 HEAT:  33.55687
 CYCLE:     4 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    38.635 HEAT:  32.55172
 CYCLE:     5 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    36.520 HEAT:  32.19731
 CYCLE:     6 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    11.576 HEAT:  31.60682
 CYCLE:     7 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     5.329 HEAT:  31.53556
 CYCLE:     8 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.076 HEAT:  31.52119
 CYCLE:     9 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.999 HEAT:  31.52072
 CYCLE:    10 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.777 HEAT:  31.52036
 CYCLE:    11 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.485 HEAT:  31.52005
 CYCLE:    12 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.554 HEAT:  31.51991
 CYCLE:    13 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.236 HEAT:  31.51958
 CYCLE:    14 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.307 HEAT:  31.51914
 CYCLE:    15 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.213 HEAT:  31.51718
 CYCLE:    16 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.692 HEAT:  31.51423
 CYCLE:    17 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     4.546 HEAT:  31.50099
 CYCLE:    18 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     3.235 HEAT:  31.48971
 CYCLE:    19 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.491 HEAT:  31.43984
 CYCLE:    20 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     4.346 HEAT:  31.41957
 CYCLE:    21 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     9.793 HEAT:  31.41920
 CYCLE:    22 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     2.247 HEAT:  31.37901
 CYCLE:    23 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.619 HEAT:  31.34523
 CYCLE:    24 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     8.360 HEAT:  31.27388
 CYCLE:    25 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    14.770 HEAT:  31.11766
 CYCLE:    26 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    20.506 HEAT:  30.88512
 CYCLE:    27 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    22.512 HEAT:  30.73299
 CYCLE:    28 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    24.223 HEAT:  30.03566
 CYCLE:    29 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    21.588 HEAT:  29.41420
 CYCLE:    30 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    27.137 HEAT:  29.10683
 CYCLE:    31 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    28.424 HEAT:  28.50239
 CYCLE:    32 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    29.074 HEAT:  27.92324
 CYCLE:    33 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    26.276 HEAT:  26.69235
 CYCLE:    34 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    17.056 HEAT:  25.86314
 CYCLE:    35 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    16.267 HEAT:  25.50047
 CYCLE:    36 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    14.605 HEAT:  25.25235
 CYCLE:    37 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    10.984 HEAT:  25.02544
 CYCLE:    38 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     6.194 HEAT:  24.81380
 CYCLE:    39 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     3.690 HEAT:  24.77234
 CYCLE:    40 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.458 HEAT:  24.75646
 CYCLE:    41 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.644 HEAT:  24.75272
 CYCLE:    42 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.384 HEAT:  24.75200
 CYCLE:    43 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.350 HEAT:  24.75137
 CYCLE:    44 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.751 HEAT:  24.74994
 CYCLE:    45 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.096 HEAT:  24.74767
 CYCLE:    46 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.883 HEAT:  24.74350
 CYCLE:    47 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.784 HEAT:  24.73740
 CYCLE:    48 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.218 HEAT:  24.73407
 CYCLE:    49 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.447 HEAT:  24.72747
 CYCLE:    50 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.035 HEAT:  24.72320
 CYCLE:    51 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.022 HEAT:  24.72067
 CYCLE:    52 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.929 HEAT:  24.71785
 CYCLE:    53 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.623 HEAT:  24.71291
 CYCLE:    54 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.889 HEAT:  24.71257
 CYCLE:    55 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.311 HEAT:  24.71122
 CYCLE:    56 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.458 HEAT:  24.71092
 CYCLE:    57 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.527 HEAT:  24.71054
 CYCLE:    58 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.390 HEAT:  24.70991
 CYCLE:    59 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.294 HEAT:  24.70929
 CYCLE:    60 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     2.100 HEAT:  24.71089
 CYCLE:    61 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.364 HEAT:  24.70854
 CYCLE:    62 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.208 HEAT:  24.70862
 CYCLE:    63 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.203 HEAT:  24.70850
 CYCLE:    64 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.200 HEAT:  24.70816
 CYCLE:    65 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.211 HEAT:  24.70798
 CYCLE:    66 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.298 HEAT:  24.70624
 CYCLE:    67 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     4.463 HEAT:  24.71105
 CYCLE:    68 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.608 HEAT:  24.70393
 CYCLE:    69 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.428 HEAT:  24.70062
 CYCLE:    70 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.492 HEAT:  24.69871
 CYCLE:    71 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.457 HEAT:  24.69723
 CYCLE:    72 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.702 HEAT:  24.69581
 CYCLE:    73 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.219 HEAT:  24.69583
 CYCLE:    74 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.297 HEAT:  24.69524
 CYCLE:    75 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.854 HEAT:  24.69445
 CYCLE:    76 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.461 HEAT:  24.69411
 CYCLE:    77 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.471 HEAT:  24.69360
 CYCLE:    78 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.327 HEAT:  24.69278
 CYCLE:    79 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.685 HEAT:  24.69240
 CYCLE:    80 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.502 HEAT:  24.69175
 CYCLE:    81 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     2.595 HEAT:  24.69471
 CYCLE:    82 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     2.123 HEAT:  24.69199
 CYCLE:    83 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     2.332 HEAT:  24.69141
 CYCLE:    84 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     4.486 HEAT:  24.69819
 CYCLE:    85 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.295 HEAT:  24.68889
 CYCLE:    86 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.567 HEAT:  24.68885
 CYCLE:    87 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.127 HEAT:  24.68775
 CYCLE:    88 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.267 HEAT:  24.68732
 CYCLE:    89 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     2.909 HEAT:  24.68977
 CYCLE:    90 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.299 HEAT:  24.68639
 CYCLE:    91 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.346 HEAT:  24.68601
 CYCLE:    92 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.407 HEAT:  24.68573
 CYCLE:    93 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.411 HEAT:  24.68544
 CYCLE:    94 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.344 HEAT:  24.68482
 CYCLE:    95 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.477 HEAT:  24.68524
 CYCLE:    96 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.104 HEAT:  24.68479
 CYCLE:    97 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.045 HEAT:  24.68472
 CYCLE:    98 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.252 HEAT:  24.68462
 CYCLE:    99 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.097 HEAT:  24.68449
 CYCLE:   100 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.049 HEAT:  24.68448
 CYCLE:   101 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.063 HEAT:  24.68441
 CYCLE:   102 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.164 HEAT:  24.68414
 CYCLE:   103 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.040 HEAT:  24.68430
 CYCLE:   104 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.041 HEAT:  24.68426
 CYCLE:   105 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.040 HEAT:  24.68428
 CYCLE:   106 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.020 HEAT:  24.68428
 CYCLE:   107 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.026 HEAT:  24.68429
 CYCLE:   108 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.470 HEAT:  24.68442
 CYCLE:   109 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.028 HEAT:  24.68429
 CYCLE:   110 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.022 HEAT:  24.68429
 CYCLE:   111 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.019 HEAT:  24.68428
 CYCLE:   112 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.127 HEAT:  24.68431
 CYCLE:   113 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.062 HEAT:  24.68430


 HEAT OF FORMATION IS ESSENTIALLY STATIONARY

 -------------------------------------------------------------------------------
PM6 GNORM=0.001 XYZ UHF CHARGE=0 Triplet

 Mopac file generated by Gabedit


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2016 (Version: 17.163L)
                                                       Fri Jun  1 14:40:03 2018
                                                       No. of days remaining =  11




          FINAL HEAT OF FORMATION =         24.68430 KCAL/MOL =     103.27910 KJ/MOL


          TOTAL ENERGY            =       -536.02270 EV
          ELECTRONIC ENERGY       =      -1116.16133 EV  POINT GROUP:     Cs  
          CORE-CORE REPULSION     =        580.13863 EV
          COSMO AREA              =         81.61 SQUARE ANGSTROMS
          COSMO VOLUME            =         55.68 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.06199
          IONIZATION POTENTIAL    =          7.577118 EV
          ALPHA SOMO LUMO (EV)    =         -7.577  3.256
          BETA  SOMO LUMO (EV)    =        -10.113 -0.819
          NO. OF ALPHA ELECTRONS  =          9
          NO. OF BETA  ELECTRONS  =          7
          MOLECULAR WEIGHT        =         35.0456

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     6    H     3     4.67884
            H     3    H     7     1.60659
            H     3    O     1     0.06092


          SCF CALCULATIONS        =        130
 
          WALL-CLOCK TIME         =          0.137 SECONDS
          COMPUTATION TIME        =          0.140 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          0.67621976  *  -1.05617698  *   0.17399899  *
     2       H          1.44866558  *  -0.44195251  *   0.62681043  *
     3       H         -0.05206885  *  -1.03075135  *   0.77151181  *
     4       N          3.52556529  *   0.69056881  *   0.91047500  *
     5       H          2.28290869  *   0.13508780  *   0.95843946  *
     6       H          3.94809441  *   1.23733377  *   1.63525719  *
     7       H          4.11721264  *   0.57787324  *   0.11007452  *

                             CARTESIAN COORDINATES

   1    O        0.676219756    -1.056176978     0.173998987
   2    H        1.448665577    -0.441952509     0.626810431
   3    H       -0.052068847    -1.030751346     0.771511808
   4    N        3.525565294     0.690568812     0.910474998
   5    H        2.282908692     0.135087798     0.958439462
   6    H        3.948094413     1.237333771     1.635257190
   7    H        4.117212642     0.577873238     0.110074516


           Empirical Formula: H5 N O  =     7 atoms



      MOLECULAR POINT GROUP   :   Cs  


          ALPHA EIGENVALUES  
 -31.15314 -28.26230 -18.55056 -18.26477 -15.43349 -14.01513 -12.94746 -10.50771
  -7.57712   3.25569   4.92873   5.62959   7.27669


           BETA EIGENVALUES 
 -28.69973 -27.36188 -17.66458 -17.33642 -14.06686 -12.18546 -10.11319  -0.81935
   2.37905   4.65385   5.95448   6.62122   8.01369


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          O          -0.723705        6.7237     1.81803     4.90567
    2          H           0.287243        0.7128     0.71276
    3          H           0.286516        0.7135     0.71348
    4          N          -0.169969        5.1700     1.65290     3.51707
    5          H          -0.044469        1.0445     1.04447
    6          H           0.176933        0.8231     0.82307
    7          H           0.187450        0.8125     0.81255
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     3.273     2.622     1.863     4.589
 HYBRID         0.594     0.433     0.254     0.777
 SUM            3.866     3.055     2.117     5.363


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  O   1.81803   1.44301   1.83103   1.63163
    2  H   0.71276
    3  H   0.71348
    4  N   1.65290   1.29223   1.07820   1.14665
    5  H   1.04447
    6  H   0.82307
    7  H   0.81255


                    (SZ)    =    1.000000
                    (S**2)  =    2.013720


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  O  0.2054  0.017  0.060  0.062  0.066
    2  H -0.0151 -0.015
    3  H -0.0111 -0.011
    4  N  1.2922  0.178  0.197  0.707  0.210
    5  H  0.5206  0.521
    6  H  0.0101  0.010
    7  H -0.0020 -0.002

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.14 SECONDS

 == MOPAC DONE ==
